api/v1alpha1/databaseuserreference_types.go

```diff
@@ -42,6 +42,10 @@ type DatabaseUserReferenceStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+//+kubebuilder:printcolumn:name="Cluster name",type=string,JSONPath=`.spec.databaseCluster.name`
```

Comment:
What happens if we specify `.spec.databaseCluster` here rather than just the name? It would be nice to see the type as well, but I'm not sure it deserves its own column. Wondering if it will pretty-print it in a reasonable way if we just tell it to print the whole object reference.

Reply:
This is what it looks like without name 
```
 NAME                 AGE     CLUSTER NAME                                                                                            USERNAME   ROLE
my-app-db-user-ref   3d17h   {"apiGroup":"databases.digitalocean.com","kind":"DatabaseClusterReference","name":"my-referenced-db"}   doadmin    primary
```

Reply:
Ah, that's gross. Let's stick with what's already in this PR.